lib/livebook.ex

```diff
@@ -104,7 +104,7 @@ defmodule Livebook do
         config :livebook, authentication_mode: :password, password: password
 
       Livebook.Config.boolean!("LIVEBOOK_TOKEN_ENABLED", true) ->
-        config :livebook, token: Livebook.Utils.random_id()
+        config :livebook, authentication_mode: :token, token: Livebook.Utils.random_id()
```

Comment:
oh, it makes sense now, because we define `true` as default value. I was trying to use it with the `dev` env but it was starting as a `disabled` auth mode